### ISSUE TITLE
Pride's Mirror Access

### ISSUE THREAD

**Issue by EmeraldCandy** [ScorpioStation/ScorpioStation#269]
**Issue Description**:
Prides mirror does not have admin-only races blacklisted for use. its possible the magic mirror has this problem as well

**Why is this bad/What are the consequences**:
Players should not have access to all of the races granted by the game

**Steps to reproduce the problem**:
Use the pride's mirror via the lavaland spawn

**Comment by Kiyahitayika** [comment]
Hey! I know where this code lives!

**Comment by Kiyahitayika** [comment]
Linking other issues w/ mirrors and Pride's Mirror:
[Pride Mirror Teleport Issue](https://github.com/ScorpioStation/ScorpioStation/issues/138)
[Using a Mirror after Changing Species Issue](https://github.com/ScorpioStation/ScorpioStation/issues/186)

**Comment by Kiyahitayika** [comment]
@blinkdog Is Scorpio continue to use Paradise as the upstream? The _mirror.dm_ file in Scorpio's codebase is currently using NanoUI. Before altering any code related to the mirrors (regular mirror, Pride's mirror, magic mirror), I recommend that Scorpio consider updating fully to TGUI. In regards specifically to the mirror code, the pull request is #14648 on Paradise's Github.

**Comment by blinkdog** [comment]
> @blinkdog Is Scorpio continuing to use Paradise as the upstream? The _mirror.dm_ file in Scorpio's codebase is currently using NanoUI. Before altering any code related to the mirrors (regular mirror, Pride's mirror, magic mirror), I recommend that Scorpio consider updating fully to TGUI. In regards specifically to the mirror code, the pull request is #14648 on Paradise's Github.

Yes, we still pull code from Paradise upstream; I just haven't done an upstream merge in awhile.
Sounds like it's about time again...
